query-engine/query-engine-node-api/Cargo.toml

```diff
@@ -23,8 +23,8 @@ psl.workspace = true
 sql-connector = { path = "../connectors/sql-query-connector", package = "sql-query-connector" }
 prisma-models = { path = "../prisma-models" }
 
-napi = { version = "=2.9.1", default-features = false, features = ["napi8", "tokio_rt", "serde-json"] }
-napi-derive = "=2.9.1"
+napi = { version = "2.10.4", default-features = false, features = ["napi8", "tokio_rt", "serde-json"] }
+napi-derive = "2.9.3"
```

Comment:
Any reason why the `=` before `napi`'s version was removed? I know they technically follow semantic versioning, but I think it'd be safer to refer to a strict version for such a core dependency

Reply:
`=` was introduced back when we pinned the dependency to avoid auto-updating it with `cargo update`. We'll still use fixed version for all builds thanks to lockfile, removing `=` just allows to bump in more easily when we need to.